public/lobby.html

```diff
@@ -37,6 +38,25 @@ <h2>Settings bar</h2>
         <div> only viewable to host</div>
         <button style="border: 1px solid black; background-color: lightgrey; width: 100px; text-align:center;" type="button" id="button-send" onclick="location.href = 'playing.html';">Start Game</button>
     </div>
+
+    <div class="chat-container">
+        <header class="chat-header">
+            <h1> Lobby Chat </h1>
+        </header>
+        <main class="chat-main">
+            <div class="chat-messages"></div>
+        </main>
+        <div class="chat-form-container">
+            <form id="chat-form">
+                <input id="msg" type="text" placeholder="Enter Message" required autocomplete="off" maxlength="60" />
+            </form>
+        </div>
+    </div>
```

Comment:
You will need to look at the backend to figure out how to properly create socket rooms and have players join them. Take a look at `public/js/chat.js` and `public/js/save.js` to see how the players and rooms are created. Specifically, try to understand this section in `chat.js`:
```js
// Store player info locally
playerSave(username, room);

// Join chatroom
const info = JSON.parse(localStorage.getItem('user-info'));
socket.emit('joinRoom', info);
```